src/replication.c

```diff
@@ -190,6 +198,77 @@ void rebaseReplicationBuffer(long long base_repl_offset) {
     }
 }
 
+/* Replication: Primary side - connections association.
+ * On rdb-connection sync, connection association is used to keep replication data in
+ * the backlog until the replica requests PSYNC. Association happens in two forms,
+ * if there's an existing buffer block at the fork time, the replica is attached
+ * to the tail, if there is no tail, the replica will be attached when a new
+ * buffer block is created (see the Retrospect function below).
+ * Replica rdb client id is used as a unique key for the association.
+ * On COB overrun, association is deleted and the RDB connection
+ * is dropped.
+ */
```

Comment:
Added a bit more structure to the comments. WDYT?

```suggestion
/* Replication: Primary side - connections association.
 * During RDB connection sync, association is used to keep replication data
 * in the backlog until the replica requests PSYNC. 
 * Association occurs in two forms:
 * 1. If there's an existing buffer block at fork time, the replica is attached to the tail.
 * 2. If there's no tail, the replica is attached when a new buffer block is created 
 *    (see the Retrospect function below).
 * The replica RDB client ID is used as a unique key for this association.
 * If a COB overrun occurs, the association is deleted and the RDB connection is dropped. */
```

Reply:
Yes, this documentation is clearer